apps/OpenSpace/ext/launcher/resources/qss/launcher.qss

```diff
@@ -61,6 +61,17 @@ LauncherWindow  QComboBox#config:disabled {
   color: rgb(225, 225, 225);
 }
 
+LauncherWindow  QMenu#newprofile {
+  background: rgb(50, 50, 50);
+  min-width: 8em;
+  max-width: 8em;
+  color: rgb(255, 255, 255);
```

Comment:
```suggestion
  color: white;
```
to be clearer when it's pure white and when it's a bright gray?   maybe we should avoid RGB values completely, really

Reply:
used white and black where we can.

Unfortunately, QSS doesn't have a way to do that except for grafting on some text-replacement, which feels iffy to do.

https://stackoverflow.com/questions/10898399/using-variables-in-qt-stylesheets